.bumpversion.cfg

```diff
@@ -1,6 +1,5 @@
 [bumpversion]
-current_version = 0.4.5-dev1
-commit = True
+current_version = 0.4.6-dev0
```

Comment:
Hey any reason we're bumping the minor version? We haven't released `0.4.5`.

You can bump the `dev-1` by doing `bump2version build` rather `patch`